tests/Backend/Exasol/TableDefinitionOperationsTest.php

```diff
@@ -53,6 +54,89 @@ private function createTableDefinition()
         return $this->_client->createTableDefinition($bucketId, $data);
     }
 
+    public function testDataPreviewForTableDefinitionWithDecimalType()
+    {
+        $bucketId = $this->getTestBucketId(self::STAGE_IN);
+
+        $tableDefinition = [
```

Comment:
imho bych trochu rozsiril ty typy co tam zkousime. Dal bych jednoho zastupce z kazdeho BaseTypu a pak vsechny divnotypy co identifikujeme jako string (binary, unique, money...)

Reply:
sorry, strcil jsem to k EXA ale melo to jit k Synapse tride. Ale to je ve vysledku jedno